src/lib/lib3index/cholesky.cc

```diff
@@ -17,7 +17,7 @@
  * with this program; if not, write to the Free Software Foundation, Inc.,
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  *
- *@END LICENSE
+ * @END LICENSE
  */
```

Comment:
@kannon92 Please look at the full diff, the entire header has been changed. Upper and lower tag, date, etc.